erlang/api_cowboy/Dockerfile

```diff
@@ -0,0 +1,18 @@
+# Usar una imagen base de Erlang
```

Comment:
A dockerfile should be useonly to deploy your application to production, in this case, you are using it to run your project locally and that's not the ideal. The only file that should run your project locally is your compose file.

I recommend the following:

1. Change the Dockerfile into a release build, it shold run `rebar3 release`, copy the generated app to a specific dir, change the base image to one without erlang, and run the project with the generated script.
2. In your docker-compose.yml file, run the project without the `build: .` param, you can check `docker-compose-local.yaml` on xmpp for reference or ask me directly if you have any question about this.